script.js

```diff
@@ -0,0 +1,432 @@
+$(document).ready(function () {
+
+    var checkIfSpell = true;
+    var classCheck = "";
+    //get all the inputs from the localstorage
+    if (localStorage.length != 0) {
+        //hit point local storage
+        if (localStorage.getItem("hitPoint") != "undefined") {
+            var hitPoint = localStorage.getItem("hitPoint", hitPoint);
+            $("#hp").val(hitPoint);
+
+        }
+        //armorclass local storage
+        if (localStorage.getItem("armorClass") != "undefined") {
+            var armorClass = localStorage.getItem("armorClass", armorClass);
+            $("#ac").val(armorClass);
+        }
+        //speed local storage
+        if (localStorage.getItem("speed") != "undefined") {
```

Comment:
You probably should have created a master object which contained all of these properties so that you could have just saved/retrieved the information with a single call.  The object would have been something like 

```
var character = {
   hitPoint: hitPoint,
   armorClass: armorClass,
  ...
```

and so on
```